pyci/rdm/algorithms.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np 
+from abc import ABCMeta, abstractmethod
+class Projection(metaclass=ABCMeta):
+    
+    @abstractmethod
```

Comment:
we don't the `@abstractmethod` if all child classes will just call it as `super().__init__()`